lib/cxml/document.rb

```diff
@@ -53,7 +53,7 @@ def render
       node = CXML.builder
       node.cXML('payloadID' => payload_id, 'timestamp' => timestamp.iso8601) do |doc|
         doc.Header { |n| @header.render(n) } if @header
-        @request.render(node) if @request
+        # @request.render(node) if @request
```

Comment:
gives errors for now because we're not passing arguments to Request through Document, the way we'd expect. want to make the fix a separate pr